coremltools/converters/mil/frontend/torch/test/test_torch_conversion_api.py

```diff
@@ -264,6 +264,32 @@ def test_func_dummy(context, inputs):
         # Cleanup the test
         del _TORCH_OPS_REGISTRY.name_to_func_mapping["test_func_dummy"]
 
+    @staticmethod
+    def test_is_torch_fx_node_supported():
```

Comment:
`is_torch_fx_node_supported` is an area where we plan to invest more in the future. I see us having many more unit tests for it in the future. I think you should put this unit test in its' own class. I would call the Class `TestFxNodeSupport` and I would call this unit test `test_simple_case`.

Reply:
Sounds good! Revised as suggested